easybuild/easyconfigs/m/MCL/MCL-14.137-GCCcore-8.3.0-Perl-5.30.0.eb

```diff
@@ -0,0 +1,39 @@
+easyblock = 'ConfigureMake'
+
+name = 'MCL'
+version = '14.137'
+versionsuffix = '-Perl-%(perlver)s'
```

Comment:
@Darkless012 You can remove this easyconfig and use the existing `MCL-14.137-GCCcore-8.3.0.eb` (we stopped using the `-Perl-*` versionsuffix, since we stick to a single Perl version per toolchain generation anyway)